workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectSystemHelper.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Unity.Entities;
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static class GameObjectSystemHelper
+    {
+        public static void AddSystems(World world)
```

Comment:
Thinking about this a bit more, if we want to make the entity game object creator non static we can make users add it here. Then this sets it on the init system. That way we can create it in the worker creator monobehaviour. Can be done in a follow up PR though.